ansible/roles/ocp4-workload-openbanking-sandbox/tasks/3amp-portal-upload-layout.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: Listing all Templates available on the Developer Portal
+  uri:
+    method: GET
+    url: "{{ _3scale_admin_url }}/admin/api/cms/templates.xml?per_page=100"
+    validate_certs: "{{ obsandbox_validate_certs|d(false)|bool }}"
+    headers:
+      Authorization: "{{ _3scale_basic_auth_token }}"
+    return_content: yes
+    status_code: 200
+  register: _3scale_portal_templates_list
+  until: _3scale_portal_templates_list.content is defined
+
+- name: Querying id for Layout "{{ _layout_system_name }}"
+  xml:
+    xmlstring: '{{ _3scale_portal_templates_list.content }}'
+    xpath: "//templates/layout[system_name='{{ _layout_system_name }}']/id"
+    content: text
+  ignore_errors: yes
+  register: _xpath_layouts
+
+- name: Retrieve the content for Layout "{{ _layout_system_name }}"
+  set_fact:
+    _layout_content: "{{ lookup('url', _layout_location, split_lines=False) }}"
+
+- name: Update pre-exising Layout "{{ _layout_system_name }}"
+  when:
+    - _xpath_layouts is defined
+    - _xpath_layouts.matches is defined
+    - _xpath_layouts.matches | length | int > 0
+  block:
+
+  - set_fact: _layout_id="{{ _xpath_layouts.matches[0].id }}"
+
+  - uri:
```

Comment:
All tasks should include `name`.

Reply:
resolved (sett commits up to `e74e328fd2d313d21b1d10138b5dbee3d2d6c2ab`)